### ISSUE TITLE
Allow other users to tag? 

### ISSUE THREAD

**Issue by acjay** [bluesky-social/proposals#4]
The original poster might not be aware of hashtags that would connect their post to interested people. It might be nice to let other people add tags, either unilaterally or by proposing them.

**Comment by davekasten** [comment]
Just my two cents as a user (not affiliated with Bluesky!  Not that much of a github user! Tell me if I'm breaking social norms here since my other github projects were culturally weird!) 

I am not in principle opposed to this, particularly if it's a "proposing" instead of "unilaterally" but think it could be _very_ easy to use that as a vector for harassment.  

I suspect some of this might be mitigated by the ways in which those tags are applied, and how much it feels like it's being pushed into the original poster's face or not.

**Comment by acjay** [comment]
I think you're right. If you want to unilaterally tag, you can quote or reply.

**Comment by pfrazee** [comment]
Yeah I think this might be a case where quote posting or replying is the move. I cant think of a way to suggest tags that isn't just spammy

**Comment by bnewbold** [comment]
I think some form of generic annotations or lists could make sense. The key thing to reduce the risk of this being a harassment vector is to not have the additional hashtags display without some relationship or reason to trust the additional tags.

There is a close analogy to delicious, pinboard, and other link tagging sites which allow anybody to tag any web resource. Also, if I recall correctly, Flickr, which had a somewhat nuanced system of adding other folks' photos to shared topic groups; for large groups this also involved asking for consent first.
